settings.gradle

```diff
@@ -63,7 +63,7 @@ gradleEnterprise {
 	buildScan {
 		termsOfServiceUrl = "https://gradle.com/terms-of-service"
 		termsOfServiceAgree = "yes"
-		publishAlways()
+		publishAlways(providers.environmentVariable('CI').present)
```

Comment:
I missed this method, fixing in #2083.

Reply:
I'm very sorry about that. I'm not sure how it escaped local testing. Thanks for fixing it

Reply:
Nevermind :)